source/_patterns/components/navigation/nav.hbs

```diff
@@ -0,0 +1,18 @@
+<ul class="nav {{ulClass}}">
```

Comment:
I think that `ulClass` and the other variables referring a type selector are too general. Is this a case to use [styleModifier](http://patternlab.io/docs/pattern-stylemodifier.html)? #51 

If it is I can ask Brian to fix our implementation so we can use it.